### ISSUE TITLE
Error encountered (broccoli-peristent-filter) when creating a new application

### ISSUE THREAD

**Issue by stavarotti** [ember-cli/ember-cli#4761]
__Environment:__

```bash
$ ember --version
version: 1.13.6
node: 2.2.1
npm: 2.13.2
os: darwin x64
```

__Problem description:__

As of today (8/25), I get a `broccoli-peristent-filter` error when creating a new application.

__Error Log:__

```bash
$ ember new project
version: 1.13.6
installing app
  create .bowerrc
  create .editorconfig
  create .ember-cli
  create .jshintrc
  create .travis.yml
  create .watchmanconfig
  create README.md
  create app/app.js
  create app/components/.gitkeep
  create app/controllers/.gitkeep
  create app/helpers/.gitkeep
  create app/index.html
  create app/models/.gitkeep
  create app/router.js
  create app/routes/.gitkeep
  create app/styles/app.css
  create app/templates/application.hbs
  create app/templates/components/.gitkeep
  create bower.json
  create config/environment.js
  create ember-cli-build.js
  create .gitignore
  create package.json
  create public/crossdomain.xml
  create public/robots.txt
  create testem.json
  create tests/.jshintrc
  create tests/helpers/resolver.js
  create tests/helpers/start-app.js
  create tests/index.html
  create tests/test-helper.js
  create tests/unit/.gitkeep
  create vendor/.gitkeep
Installing packages for tooling via npm...
npm ERR! git rev-list -n1 broccoli-peristent-filter: fatal: ambiguous argument 'broccoli-peristent-filter': unknown revision or path not in the working tree.
npm ERR! git rev-list -n1 broccoli-peristent-filter: Use '--' to separate paths from revisions, like this:
npm ERR! git rev-list -n1 broccoli-peristent-filter: 'git <command> [<revision>...] -- [<file>...]'
npm ERR! git rev-list -n1 broccoli-peristent-filter:
npm ERR! git rev-list -n1 broccoli-peristent-filter: fatal: ambiguous argument 'broccoli-peristent-filter': unknown revision or path not in the working tree.
npm ERR! git rev-list -n1 broccoli-peristent-filter: Use '--' to separate paths from revisions, like this:
npm ERR! git rev-list -n1 broccoli-peristent-filter: 'git <command> [<revision>...] -- [<file>...]'
npm ERR! git rev-list -n1 broccoli-peristent-filter:
npm ERR! git rev-list -n1 broccoli-peristent-filter: fatal: ambiguous argument 'broccoli-peristent-filter': unknown revision or path not in the working tree.
npm ERR! git rev-list -n1 broccoli-peristent-filter: Use '--' to separate paths from revisions, like this:
npm ERR! git rev-list -n1 broccoli-peristent-filter: 'git <command> [<revision>...] -- [<file>...]'
npm ERR! git rev-list -n1 broccoli-peristent-filter:
Command failed: git rev-list -n1 broccoli-peristent-filter
fatal: ambiguous argument 'broccoli-peristent-filter': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'

Error: Command failed: git rev-list -n1 broccoli-peristent-filter
fatal: ambiguous argument 'broccoli-peristent-filter': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'

    at ChildProcess.exithandler (child_process.js:195:12)
    at emitTwo (events.js:87:13)
    at ChildProcess.emit (events.js:172:7)
    at maybeClose (internal/child_process.js:763:16)
    at Socket.<anonymous> (internal/child_process.js:318:11)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at Pipe._onclose (net.js:466:12)
```

**Comment by kellyselden** [comment]
duplicate of #4760

**Comment by rwjblue** [comment]
Fixed by https://github.com/babel/ember-cli-babel/pull/56 and released as 5.1.5.
